granian/wsgi.py

```diff
@@ -16,10 +16,10 @@ def __call__(self, status: str, headers: List[Tuple[str, str]], exc_info: Any =
         self.headers = headers
 
 
-def _callback_wrapper(callback, scope_opts):
+def _callback_wrapper(callback: Callable[..., Any], scope_opts: Dict[str, Any]):
     basic_env = dict(os.environ)
     basic_env.update(
-        {
+        **{
```

Comment:
Why adding the `**` operator here?

Reply:
Good point @gi0baro. I think you are right, unpacking the dictionary is unnecessary. I did it just to fix the **pylance** bug, but I think here it is enough to explicitly specify the type for the solution. This will be fixed.